packages/client/src/object/fetchPageOrThrow.ts

```diff
@@ -28,16 +28,28 @@ import type { PageResult } from "../PageResult.js";
 import { convertWireToOsdkObjects } from "./convertWireToOsdkObjects.js";
 
 export interface SelectArg<
-  O extends ObjectOrInterfaceDefinition<any, any>,
-  L = ObjectOrInterfacePropertyKeysFrom2<O>,
+  Q extends ObjectOrInterfaceDefinition<any, any>,
+  L = ObjectOrInterfacePropertyKeysFrom2<Q>,
 > {
   select?: readonly L[];
 }
 
+export interface OrderByArg<
+  Q extends ObjectOrInterfaceDefinition<any, any>,
+  L = ObjectOrInterfacePropertyKeysFrom2<Q>,
+> {
+  orderBy?: {
+    [K in L & string]?: "asc" | "desc";
```

Comment:
Why do we need to & string here?

Reply:
Ah yeah the L typing is a little sloppy, can fix